harness.sh

```diff
@@ -238,43 +251,44 @@ chmod +x "${NODES_ROOT}/master/mine"
 tmux new-window -t $SESSION:1 -n 'mctl'
 tmux send-keys "cd ${NODES_ROOT}/master" C-m
 
-if [ "${NETWORK}" = "simnet" ]; then
-echo "Waiting for some ${NETWORK} blocks to be mined"
-sleep 2
+sleep 3
 # mine some blocks to start the chain if the active network is simnet.
+if [ "${NETWORK}" = "simnet" ]; then
 tmux send-keys "./mine 2" C-m
+echo "Mined 2 blocks"
+sleep 1
 fi
 
-if [ "${NETWORK}" = "simnet" ]; then
+tmux send-keys "./ctl livetickets"
+
 ################################################################################
 # Setup the pool wallet.
 ################################################################################
-cat > "${NODES_ROOT}/wallet/ctl" <<EOF
+cat > "${NODES_ROOT}/mwallet/ctl" <<EOF
 #!/bin/sh
-dcrctl -C ../dcrwctl.conf --wallet \$*
+dcrctl -C ../dcrmwctl.conf --wallet \$*
 EOF
-chmod +x "${NODES_ROOT}/wallet/ctl"
+chmod +x "${NODES_ROOT}/mwallet/ctl"
 
-tmux new-window -t $SESSION:2 -n 'wallet'
-tmux send-keys "cd ${NODES_ROOT}/wallet" C-m
-tmux send-keys "dcrwallet -C wallet.conf --create" C-m
-echo "Creating ${NETWORK} wallet"
-sleep 2
+tmux new-window -t $SESSION:2 -n 'mwallet'
+tmux send-keys "cd ${NODES_ROOT}/mwallet" C-m
+tmux send-keys "dcrwallet -C mwallet.conf --create" C-m
+echo "Creating ${NETWORK} master wallet"
+sleep 1
 tmux send-keys "${WALLET_PASS}" C-m "${WALLET_PASS}" C-m "n" C-m "y" C-m
-echo "Setting wallet passphrase"
 sleep 1
-tmux send-keys "${WALLET_SEED}" C-m C-m
-tmux send-keys "dcrwallet -C wallet.conf" C-m
+tmux send-keys "${MASTER_WALLET_SEED}" C-m C-m
+tmux send-keys "dcrwallet -C mwallet.conf " C-m # --debuglevel=warn
 
-################################################################################
-# Setup the pool wallet's dcrctl (wctl).
-################################################################################
-echo "Waiting for nodes to be synced"
-sleep 6
+# ################################################################################
+# # Setup the pool wallet's dcrctl (wctl).
+# ################################################################################
+sleep 5
```

Comment:
As I mentioned on the previous review, this needs to be increased to `sleep 10` in order to work reliably on my machine

Reply:
noted, thanks.